backpack/custom_module/permute.py

```diff
@@ -8,16 +8,18 @@
 class Permute(Module):
     """Module to permute a tensor."""
 
-    def __init__(self, *dims: Any, batch_axis: int = 0):
+    def __init__(self, *dims: Any, init_transpose: bool = False, batch_axis: int = 0):
         """Initialization.
 
         Args:
             dims: The desired ordering of dimensions.
+            init_transpose: If transpose parameters are provided. Default: False.
```

Comment:
Not entirely sure what this is doing? Support for permutations and transposition?

Reply:
Yes. Added an explanation.